build.gradle

```diff
@@ -1,37 +1,116 @@
+// TODO: We can't seem to use the nexus-staging plugin since
+// our package is put in with other softwareamazon repos
+// and the plugin can't automatically find ours.  For now
+// we'll just use the UI. An automatic solution would be nice.
+
+plugins {
+    id 'io.spring.dependency-management' version '1.0.7.RELEASE'
+}
+
 apply plugin: 'java'
+apply plugin: 'maven'
```

Comment:
As of Gradle 4.8 (July 2018), the 'maven' plugin is deprecated.  You should use the 'maven-publish' plugin instead.